src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate: function (operation, number = 0) {
```

Comment:
Instead of using the `function` keyword, you can define the `operate` method using the shorthand syntax to be consistent with the other methods.